plugins/module_utils/gandi_livedns_api.py

```diff
@@ -50,7 +51,12 @@ def _build_error_message(self, module, info):
         return s
 
     def _gandi_api_call(self, api_call, method='GET', payload=None, error_on_404=True):
-        headers = {'Authorization': 'Apikey {0}'.format(self.api_key),
+        authorization_header = (
+            'Bearer {}'.format(self.personal_access_token)
+            if self.personal_access_token
+            else 'Apikey {}'.format(self.api_key)
```

Comment:
Hmm, I missed this Python 2.6 problem reported by the sanity checks:
```suggestion
            'Bearer {0}'.format(self.personal_access_token)
            if self.personal_access_token
            else 'Apikey {0}'.format(self.api_key)
```

Reply:
Nice catch, also missed it. I didn't know Python 2.6 was still a thing (and didn't even know it didn't support automatic indexing with `format`). Should be good now.

Reply:
Hmm, actually we don't support Python 2.6 anymore (I'm still getting used to that; ansible-core 2.12 was the last to support it, and with community.general 9.0.0 the minimum supported version will be ansible-core 2.13), but for some reason there's an extra pylint rule in ansible-core 2.13's `pylint` sanity check that requires this. 
```
ERROR: plugins/module_utils/gandi_livedns_api.py:55:12: ansible-format-automatic-specification: Format string contains automatic field numbering specification (100%)
ERROR: plugins/module_utils/gandi_livedns_api.py:57:17: ansible-format-automatic-specification: Format string contains automatic field numbering specification (100%)
```
It *might* be that someone forgot to remove that rule (since it was mainly for Python 2.6 compatibility) after Python 2.6 support was dropped, but well... :shrug: I guess having the indexes there doesn't hurt either :)